cms/tests/test_apphooks.py

```diff
@@ -307,7 +307,7 @@ def test_apphook_permissions_preserves_view_name(self):
         view_names = (
             ('sample-settings', 'sample_view'),
             ('sample-class-view', 'ClassView'),
-            ('sample-class-based-view', 'view' if not (DJANGO_3 or DJANGO_2_2) else 'ClassBasedView' ),
+            ('sample-class-based-view', 'ClassBasedView' ),
```

Comment:
We still support older versions of django, so I feel like this still needs conditional logic.

Reply:
@marksweb I think the PR is trying to achieve not having to have the Django version conditionals in custom code (or tests) but to always forward the view class name as presented by Django when using CMS decorators independently of the underlying Django version. @Will-Hoey Am I correct here?

Reply:
@fsbraun That is correct yes!